app/services/rebate_forms_service.rb

```diff
@@ -8,66 +8,35 @@ def initialize(rebate_form_attributes)
     @rebate_form_attributes = rebate_form_attributes
   end
 
-  def update!
+  def create!
     council = find_council!
-    property = create_or_update_property!(council)
-    rebate_form = create_or_update_rebate_form!(property)
-    create_or_update_rates_bill!(property)
-    rebate_form.calc_rebate_amount!
-    raise Error if rebate_form.rebate.blank?
+    property = find_or_create_property!(council)
+    rebate_form = create_rebate_form!(property)
 
     rebate_form
```

Comment:
This isn't needed here - `create-rebate_form!` returns the RebateForm, so you don't need the variable on line 14 or to return it on line 16.